JuiceMaker/JuiceMaker/Controller/ViewController.swift

```diff
@@ -7,9 +7,110 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    @IBOutlet var fruitLabelCollection: [UILabel]!
+    private let juiceMaker = JuiceMaker()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        updateFruitsStock()
+    }
+    
+    private func updateFruitsStock() {
+        var index:Int = .zero
```

Comment:
```suggestion
        var index: Int = .zero
```
요기 살짝 띄워주면 더 좋을 것 같아요 +_+

Reply:
앜 감사합니다~😉